Automattic/wp-calypso
# Signup: Remove unused rebrand-cities flow

#### Changes proposed in this Pull Request

The `rebrand-cities` signup flow is unused (see the `?32169/2021-11-15/to/2021-12-15` tracks funnel).

This PR removes it, along with some related components, utilities, styles and illustrations.

#### Testing instructions

* Smoke test signup and checkout.
* Verify the removed code is not in use.
* Verify tests pass.
